test/test_dagnav.py

```diff
@@ -743,3 +851,137 @@ def test_surface_load_file(request):
     # Non-STL file should not be allowed
     with pytest.raises(ValueError):
         model.create_surface(filename='badgers.exe')
+
+def test_surface_sense_runtime_error():
+    """Test senses returns default when tag is missing."""
+    model = pydagmc.Model()
+    surf = model.create_surface(global_id=1)
+    try:
+        model.mb.tag_delete(model.surf_sense_tag, (surf.handle,))
+    except RuntimeError:
```

Comment:
Similar situation to the name tag for groups above. It should be safe to assume that this tag exists.